src/util.js

```diff
@@ -35,10 +35,10 @@ var util = {
     }
   },
 
-  globalContext: (typeof(window) !== 'undefined' ? window : global),
+  globalContext: (typeof(window) !== 'undefined' ? window : (typeof self === 'object' ? self : global)),
 
   getGlobalContext () {
-    return (typeof(window) !== 'undefined' ? window : global);
+    return (typeof(window) !== 'undefined' ? window : (typeof self === 'object' ? self : global));
```

Comment:
What is this new change for?

Reply:
This sets the global context, when run in a Service Worker, where the global variable is called 'self'.